src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -84,6 +84,11 @@ limitations under the License.
             <include cluster="Ethernet Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="WiFi Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
             <include cluster="Thread Network Diagnostics" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="ICD Management" clients="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>IDLE_MODE_DURATION</requireAttribute>
+                <requireAttribute>ACTIVE_MODE_DURATION</requireAttribute>
+                <requireAttribute>ACTIVE_MODE_THRESHOLD</requireAttribute>
```

Comment:
Why do we need those?  Are those not marked as required in the cluster spec?  I think `requireAttribute` is for device-level overrides of the cluster specs....

Reply:
@bzbarsky-apple Based on the OnOff light device type, everything that is mandatory is indicated with the `required...`.

Reply:
I think the OnOff device type XML is wrong, fwiw.  But maybe I don't understand how ZAP processes these things....

Reply:
Basically: Ideally the XML would reflect the spec here, and the spec does not have these requirements in the _device type_.

Reply:
removed the attributes